edgecompute/examples/stream/stream-concat/main.js

```diff
@@ -0,0 +1,46 @@
+/*
+(c) Copyright 2022 Akamai Technologies, Inc. Licensed under Apache 2 license.
+
+Purpose:  Concatenate multiple subrequests into a single output stream
+
+*/
+
+import { httpRequest } from 'http-request';
+import { createResponse } from 'create-response';
+import { TransformStream } from 'streams';
+
+async function concatSubrequestsToStream(urls, writableStream) {
+
+  // Begin requests for all URLs, generating a list of promises
+  const responsePromises = urls.map(url => httpRequest(url));
```

Comment:
Should the save request headers be included here, just so the example shows including the request headers?